NullPointerException when doing a query

```
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because "preview" is null 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.eclipse.openvsx.LocalRegistryService.lambda$query$1(LocalRegistryService.java:338) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:214) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1716) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:635) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:291) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:652) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:658) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:663) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.eclipse.openvsx.LocalRegistryService.query(LocalRegistryService.java:340) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.lang.reflect.Method.invoke(Method.java:565) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:360) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:724) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.eclipse.openvsx.LocalRegistryService$$SpringCGLIB$$0.query(<generated>) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.eclipse.openvsx.RegistryAPI.getQuery(RegistryAPI.java:1029) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at java.base/java.lang.reflect.Method.invoke(Method.java:565) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:258) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:191) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:991) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:896) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1089) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:979) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014) 
[open-vsx-org-production-6c5cf98796-svvh8 open-vsx-org] 	... 170 common frames omitted 
```

--- a/server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java
+++ b/server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java
@@ -252,7 +252,7 @@ public class LocalRegistryService implements IExtensionRegistry {
         list.setPostUrl(createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(), "review"));
         list.setDeleteUrl(createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(), "review", "delete"));
         list.setReviews(repositories.findActiveReviews(extension)
-                .map(extReview -> extReview.toReviewJson())
+                .map(ExtensionReview::toReviewJson)
                 .toList());
         return list;
     }
@@ -342,6 +342,10 @@ public class LocalRegistryService implements IExtensionRegistry {
                     var latestPreRelease = latestPreReleases.get(getLatestVersionKey(ev));
                     var reviewCount = reviewCounts.getOrDefault(ev.getExtension().getId(), 0L);
                     var preview = previewsByExtensionId.get(ev.getExtension().getId());
+                    if (preview == null) {
+                        logger.warn("Unable to determine if extension {} is a preview version", NamingUtil.toLogFormat(ev));
+                        preview = Boolean.FALSE;
+                    }
                     var extensionVersionStrings = versionStrings.get(ev.getExtension().getId());
                     var fileResources = fileResourcesByExtensionVersionId.getOrDefault(ev.getId(), Collections.emptyList());
                     return toExtensionVersionJson(ev, latest, latestPreRelease, reviewCount, preview, extensionVersionStrings, targetPlatform, fileResources, membershipsByNamespaceId);
@@ -383,6 +387,10 @@ public class LocalRegistryService implements IExtensionRegistry {
                     var latestPreRelease = latestPreReleases.get(getLatestVersionKey(ev));
                     var reviewCount = reviewCounts.getOrDefault(ev.getExtension().getId(), 0L);
                     var preview = previewsByExtensionId.get(ev.getExtension().getId());
+                    if (preview == null) {
+                        logger.warn("Unable to determine if extension {} is a preview version", NamingUtil.toLogFormat(ev));
+                        preview = Boolean.FALSE;
+                    }
                     var fileResources = fileResourcesByExtensionVersionId.getOrDefault(ev.getId(), Collections.emptyList());
                     var globalLatest = addAllVersions ? latestGlobalVersions.get(ev.getExtension().getId()) : null;
                     var globalLatestPreRelease = addAllVersions ? latestGlobalPreReleases.get(ev.getExtension().getId()) : null;
@@ -1121,7 +1129,7 @@ public class LocalRegistryService implements IExtensionRegistry {
         }
 
         var namespace = extVersion.getExtension().getNamespace().getId();
-        var memberships = membershipsByNamespaceId.get(namespace);
+        var memberships = membershipsByNamespaceId.getOrDefault(namespace, Collections.emptyList());
         return memberships.stream().anyMatch(m -> m.getRole().equalsIgnoreCase(NamespaceMembership.ROLE_OWNER))
                 && memberships.stream().anyMatch(m -> m.getUser().getId() == user.getId());
     }

